pkg/updater/download.go

```diff
@@ -198,6 +198,8 @@ func extractTarArchive(archivePath string, destinationPath string, compression c
 			if err != nil {
 				return fmt.Errorf("could not create symlink: %w", err)
 			}
+		case tar.TypeLink:
+			// we currently don't support hard links in the updater
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 Praise:** ‏Nice. Should we add a todo ticket?

Reply:
We can but I would not pro-actively work on this until it blocks something given the added complexity